drivers/ufs/ufs.c

```diff
@@ -533,6 +537,8 @@ static void ufs_query(uint8_t op, uint8_t idn, uint8_t index, uint8_t sel,
 		       (void *)(utrd.resp_upiu + sizeof(query_resp_upiu_t)),
 		       size);
 		break;
+	default:
+		break;
```

Comment:
As with the others, I think good to add a comment indicating that this expected to be empty.